packages/api-explorer/src/prop-types/doc.js

```diff
@@ -0,0 +1,27 @@
+import PropTypes from 'prop-types';
+
+export default {
+  title: PropTypes.string.isRequired,
+  excerpt: PropTypes.string,
+  slug: PropTypes.string.isRequired,
+  type: PropTypes.string.isRequired,
+  api: PropTypes.shape({
+    method: PropTypes.string.isRequired,
+    examples: PropTypes.shape({
+      codes: PropTypes.arrayOf(
+        PropTypes.shape({
+          language: PropTypes.string.isRequired,
+          code: PropTypes.string.isRequired,
+        }),
+      ),
+    }),
+    results: PropTypes.shape({
+      codes: PropTypes.arrayOf(
+        PropTypes.shape({})
```

Comment:
An array of empty object?

Reply:
I've extracted what we already hade. Is it risky to introduce now a restriction?